src/messages/en.json

```diff
@@ -91,7 +91,18 @@
     },
     "Footer": {
         "imprint": "Legal Notice",
-        "dataProtection": "Data Protection Policy"
+        "dataProtection": "Privacy Policy",
+         "sourceCode" : "Sourcecode",
```

Comment:
Please reword to `Source`. Sounds better imo. Also, the `sourceCode` entry is missing in the German translations.

Afterwards, please run `CMD + Shift + P` -> `JSON: Sort document` action in VSCode.